### DISCUSSION TITLE
Status Bericht Woche 9

### DISCUSSION THREAD

**Discussion by CodeRider101** [SE-TINF22B2/G2-Schoolify#87] [Status Berichte]
ich möchte Sie daran erinnern, dass der letzte „Statusbericht“ in diesem Semester aus einer Statistik der Aufwände je Person und Phase des SE-Prozesses sowie der Mitarbeiterumfrage besteht.

Senden Sie mir diese Informationen bitte noch diese Woche per Mail zu (bzw. mit einem Hinweis, wo sich diese Informationen in Ihrem Repo befinden).

**Comment by laralngn** [comment]
| Person | Aufwand |
| ----------- | ----------- |
| Lara  | 12.5 Personenstunden |
| Gerrit  | 8 Personenstunden |
| Yannis  | 12  Personenstunden |
| Jere | 10 Personenstunden |
| Marvin | 11 Personenstunden |
| Til | 7.5 Personenstunden |
|∑| 61 Personenstunden + (20 Personenstunden Meeting und Planung)|


| Iteration | Aufwand |
| ----------- | ----------- |
| Iteration 1 | 1.5 Personenstunden |
| Iteration 2 | 5.5 Personenstunden |
| Iteration 3 | 12 Personenstunden |
| Iteration 4 | 25.5 Personenstunden |
| Iteration 5 | 3.5 Personenstunden |
| Iteration 6 | 7 Personenstunden |
| Iteration 7 | 4.5 Personenstunden |
| Iteration 8 | 0.5 Personenstunden |
| Iteration 9 | 1 Personenstunden |
|∑| 61 Personenstunden |



Mitarbeiterumfrage: [Link](https://github.com/SE-TINF22B2/G2-Schoolify/discussions/78)

**Reply by roland-KA** [reply]
Wie sich diese Aufwände auf die Phasen RE, Design, ..., Projektmanagement aufteilen konnte nicht mehr ermittelt werden?

**Reply by CodeRider101** [reply]
phase | effort
-- | -- 
Requirements | 31 Personenstunden
Design | 16.5 Personenstunden
Projectmanagement | 6.5 Personenstunden
Construction | 7 Personenstunden
∑ | 61 Personenstunden

**Comment by two-Gee** [comment]
Zusätzlich pro Person 4 Stunden Meetings

**Comment by CodeRider101** [comment]
# Mitarbeiterumfrage: [Link](https://github.com/SE-TINF22B2/G2-Schoolify/discussions/78)
